cmd/lncli/commands.go

```diff
@@ -122,12 +122,12 @@ var newAddressCommand = cli.Command{
 	Action: actionDecorator(newAddress),
 }
 
-func newAddress(ctx *cli.Context) error {
+func newAddress(clictx *cli.Context) error {
```

Comment:
I would name this `cliCtx`, and also do this commit first. Then we could make the other context named `ctx` without first naming it `ctxt`.

Reply:
done